File: archive/jewels-and-stones.js
const numJewelsInStones = function(jewels, stones) {
  let jMem = {}
  jewels.split('').forEach(l => {
    jMem[l] = true
  })
  let ans = 0
  for (let i = 0; i < stones.length; i++) {
    if (jMem[stones[i]]) {
      ans++
    }
  }

  return ans
};
//"aA", stones = "aAAbbbb"


console.log(numJewelsInStones("aA", "aAAbbbb"))

File: go-archive/jewels-and-stones.go
func numJewelsInStones(jewels string, stones string) int {
    set:=make(map[byte]bool); ans:=0

    for i:=0; i < len(jewels); i++ {
        set[jewels[i]] = true
    }

    for i:=0; i < len(stones); i++ {
        if set[stones[i]] {ans++}
    }
    return ans
}